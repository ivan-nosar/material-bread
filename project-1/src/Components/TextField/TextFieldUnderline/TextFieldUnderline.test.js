import React from 'react';
import TextFieldUnderline from './TextFieldUnderline';

import renderer from 'react-test-renderer';

test('TextFieldUnderline Renders', () => {
  const tree = renderer.create(<TextFieldUnderline />).toJSON();
  expect(tree).toMatchSnapshot();
});
