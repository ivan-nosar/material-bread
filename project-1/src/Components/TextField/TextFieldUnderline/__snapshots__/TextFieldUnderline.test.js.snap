// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFieldUnderline Renders 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0, 0.42)",
      "bottom": 0,
      "height": 2,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "transform": Array [
        Object {
          "scaleY": 0.5,
        },
      ],
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(33, 150, 243, 1)",
        "bottom": 0,
        "height": 2,
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "transform": Array [
          Object {
            "scaleX": 0,
          },
          Object {
            "scaleY": 0,
          },
        ],
      }
    }
  />
</View>
`;
